App.jsx

```diff
@@ -4,6 +4,7 @@ import { Router,Outlet} from '@tanstack/react-location';
 import { routes,location } from './Router';
 import { Suspense } from 'react';
 import Header from './components/Header/Header';
+import Docs from './Docs';
```

Comment:
where is this used? I don't see in this file

Reply:
Not needed so removed